.gitignore

```diff
@@ -87,6 +87,7 @@ jenkins/docker/regular-tests/Dockerfile
 ml/src-gen
 scoring/src-gen
 py/src-gen
+py-scoring/src-gen
 r/src-gen
```

Comment:
FYI, in `git-ignore` , the following pattern will ignore any subfolder named `src-gen`
```
src-gen/
```

Reply:
for confirmation (https://git-scm.com/docs/gitignore)
```
If there is a separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular .gitignore file itself. Otherwise the pattern may also match at any level below the .gitignore level.
```

Reply:
`py-scoring/src-gen` worked for me locally, but will change according to your suggestion. Thank you!

Reply:
it's just that you were listing all the `src-gen` modules, and usually you want to excluded those by default.